src/main.js

```diff
@@ -0,0 +1,22 @@
+function generateLink() {
```

Comment:
Is this even possible? Using plain javascript without a framework? :O

Reply:
I'm going to be honest, I didn't test if this works on Netscape Navigator :smile: